modular_skyrat/modules/novaya_ert/code/head.dm

```diff
@@ -1,24 +1,33 @@
+/obj/item/clothing/head/helmet/rus_helmet
+	name = "Russian L47 helmet"
+	desc = "NRI mass-produced 42-AM helmet, a thin layer of experimental alloy provides limited protection against laser and energy and... Polar bears?"
```

Comment:
```suggestion
	desc = "NRI mass-produced 42-AM helmet, a thin layer of experimental alloy provides limited protection against laser and energy and... polar bears?"
```

Reply:
Dont care too much about it, it will be reworked.
Cause we are doing it right now, many things will be changed.